raiden/transfer/mediated_transfer/target.py

```diff
@@ -243,7 +243,7 @@ def handle_inittarget_light(
                                                             receiver_light_client_address=transfer.target)
 
             secret_request_message = SecretRequest.from_event(secret_request)
-            sr_receiver_light_client_address = transfer.initiator if channel_state.both_participants_are_light_clients else None
+            sr_receiver_light_client_address = transfer.initiator
```

Comment:
This cannot be always the transfer.initiator because on payments between LC and FN the initiator can be a not registered client, therefore the client FK will fail